Makefile

```diff
@@ -95,8 +95,16 @@ release: all deps docs $(SMF_MANIFESTS)
 	$(TOP)/smf \
 	$(TOP)/npm \
 	$(RELSTAGEDIR)/$(NAME)
+	rm -rf $(RELSTAGEDIR)/$(NAME)/node_modules/eslint \
```

Comment:
OK, I thought the target `all` was the only responsible for running `npm install`, and that we could rely on that to use any command line option we want it to use. If that's not the case then `rm -rf` is not overkill.

Whitelisting could also work, but that's essentially what `npm install --production` does.